tests/unittests/CoreGraphics.drawing/CTDrawingTests.cpp

```diff
@@ -499,10 +499,18 @@ TEXT_DRAW_TEST_P(Fonts, TestFonts) {
     __DrawLoremIpsum(context, path.get(), keys, values);
 }
 
+// Segoe UI is noticeably different on ARM, but we still want to test other fonts so only remove Segoe
+#ifdef WINOBJC_DISABLE_TESTS
```

Comment:
missed the comment from code flow. Just remove Segoe UI tests.  We don't do anything fancy with the fonts themselves - just use the DWrite apis.